inventory/environ.py

```diff
@@ -217,58 +294,37 @@ def overrideEnvironmentVars(vars_scope):
     vars_scope["splunk"]["s2s"]["port"] = int(os.environ.get('SPLUNK_S2S_PORT', vars_scope["splunk"]["s2s"]["port"]))
     vars_scope["splunk"]["secret"] = os.environ.get('SPLUNK_SECRET', vars_scope["splunk"]["secret"])
     vars_scope["splunk"]["hec_token"] = os.environ.get('SPLUNK_HEC_TOKEN', vars_scope["splunk"]["hec_token"])
-    if "shc" not in vars_scope["splunk"]:
-        vars_scope["splunk"]["shc"] = {}
-    vars_scope["splunk"]["shc"]["label"] = os.environ.get('SPLUNK_SHC_LABEL', vars_scope["splunk"]["shc"]["label"])
-    vars_scope["splunk"]["shc"]["secret"] = os.environ.get('SPLUNK_SHC_SECRET', vars_scope["splunk"]["shc"]["secret"])
-    if "idxc" not in vars_scope["splunk"]:
```

Comment:
This section was removed from `overrideEnvironmentVars()`, split into its own function `getIndexerClustering()`, and moved directly into `getDefaultVars()`